packages/app/src/styles/theme/_apply-colors-light.scss

```diff
@@ -236,10 +238,31 @@
         hsl.darken(var(--bgcolor-sidebar-context),24%)
       );
 
-      .grw-pagetree-triangle-btn {
+      .grw-pagetree-triangle-btn, .grw-foldertree-triangle-btn {
         @include mixins-buttons.button-outline-svg-icon-variant($gray-400, var(--primary));
       }
     }
+
+    // bookmark
+    .grw-folder-tree-container {
+      .grw-drop-item-area, .grw-foldertree-item-container  {
+        .grw-accept-drop-item {
+          border-color: hsl.darken(var(--bgcolor-sidebar-context), 30%) !important;
+        }
+      }
+    }
+
+    .grw-bookmarks-list {
+      @include override-list-group-item-for-pagetree(
+        var(--color-sidebar-context),
+        hsl.darken(var(--bgcolor-sidebar-context),5%),
+        hsl.darken(var(--bgcolor-sidebar-context),12%),
+        hsl.lighten(var(--color-sidebar-context),10%),
+        hsl.lighten(var(--color-sidebar-context),8%),
+        hsl.darken(var(--bgcolor-sidebar-context),15%),
+        hsl.darken(var(--bgcolor-sidebar-context),24%)
+      );
+    }
```

Comment:
Is this `grw-bookmarks-list` class used?

Reply:
This class is no longer used, so I removed it